hibernate-core/src/main/java/org/hibernate/query/Query.java

```diff
@@ -855,6 +856,9 @@ default Query<R> applyLoadGraph(@SuppressWarnings("rawtypes") RootGraph graph) {
 	@Override
 	Query<R> setHibernateFlushMode(FlushMode flushMode);
```

Comment:
Is there a reason this is not deprecated?  It seems to me the intent here is to replace `FlushMode` when used in the context of a Query

Reply:
I could deprecate it if you prefer.